flink-python/pyflink/table/udf.py

```diff
@@ -147,6 +234,41 @@ def eval(self, *args):
         return self.func(*args)
 
 
+class DelegatingPandasAggregateFunction(AggregateFunction):
+    """
+    Helper pandas aggregate function implementation for lambda expression and python function.
+    It's for internal use only.
+    """
+
+    def __init__(self, func):
+        self.func = func
+
+    def get_value(self, accumulator):
+        return accumulator[0]
+
+    def create_accumulator(self):
+        return []
+
+    def accumulate(self, accumulator, *args):
+        accumulator.append(self.func(*args))
+
+
+class WrapperPandasAggregateFunction(object):
```

Comment:
Rename it as `PandasAggregateFunctionWrapper` and add some description about this class?